spec/factories/bus_stops.rb

```diff
@@ -14,11 +14,11 @@
         end
       created_at Date.yesterday
       updated_at Date.today
-        %i[bolt_on_base bus_pull_out_exists has_power solar_lighting
-          system_map_exists shelter_ada_compliance ada_landing_pad
-          state_road].each do |b|
-          send(b, 'true')
-          end
+      %i[bolt_on_base bus_pull_out_exists has_power solar_lighting
+        system_map_exists shelter_ada_compliance ada_landing_pad
+        state_road].each do |b|
```

Comment:
Probably better to state them outright:

```ruby
bolt_on_base        true
bus_pull_out_exists true
```

It seems repetitive, but that's fine. I think the advantage is that it reads how a factory is supposed to read.